scripts/run_experiment.sh

```diff
@@ -19,7 +19,7 @@ echo "Generating run: (P=${PARTICLE_NO}, N=${N}, DIR=${RUN_DIR})"
 T_START=$(date +"%s%N")
 
 ## generate jobs
-generatemc -p ${PARTICLE_NO} -j ${N} ${BASE_DIR}/input/data/ --workspace ${RUN_DIR} --scheduler_options "[--time=0:39:59 -A plgccbmc11-cpu]"
+generatemc -p ${PARTICLE_NO} -j ${N} ${BASE_DIR}/input/data/ --workspace ${RUN_DIR} --scheduler_options "[--time=0:09:59 -A plgccbmc11-cpu]"
```

Comment:
Z ciekawości: po co to? (nie ogarniam opcji schedulera)

Reply:
Jest to górne ograniczenie czasu działania - krótsze zadania mają większy priorytet. Jedynym potencjalnym problemem jest ubijanie zadań, jeśli ten czas zostanie przekroczony, ale w tym przypadku jest ok (dla większych problemów pewnie będzie trzeba zwiększać, a potencjalnie można to dodać do configu)

Reply:
Danke schon